SQL/proc_descriptions/pgtap.md

```diff
@@ -0,0 +1,207 @@
+# Notes regarding pgTap testing framework
+
+## Instalation
+1) pgTAP must be installed on a host with PostgreSQL server running; it cannot be installed remotely. If you’re using PostgreSQL in Docker, you need to install pgTAP inside the Docker container.
+    If you are using Linux, you may (depending on your distribution) be able to use you distribution’s package management system to install pgTAP. For instance, on Debian, Ubuntu, or Linux Mint pgTAP can be installed with the command:
+    `sudo apt-get install pgtap`
+    On other systems pgTAP has to be downloaded and built. First, download pgTAP from PGXN (click the green download button in the upper-right). Extract the downloaded zip file, and (at the command line) navigate to the extracted folder.
```

Comment:
The thing is I expect that our Postgres server is running on Linux server, so the functions and extensions including _pgTap_ would be there and it would be no problem to run tests. Or do you want me to update this section with windows installation in order to let Windows users to test locally?

Reply:
Documentation says "On other systems (excluding Docker & Linux) pgTAP has to be downloaded and built.". Checked out https://pgxn.org/dist/pgtap/, which also has not anything related to pre-built package

Reply:
I've got no windows machine, including VM (free virtualisers are not optimised to arm M1 chip)

Reply:
ok, we need to involve someone else then. Can you ask in the roadgraphtool chanel for help?

Reply:
Sure